src/pages/app/index.css

```diff
@@ -0,0 +1,37 @@
+@tailwind base;
```

Comment:
We can reuse this CSS across files, but it'd require a postcss setup. Can add in future, for now I was keeping it light and let TailwindCSS manage it from defaults.